README.FA.md

```diff
@@ -0,0 +1,72 @@
+* سرور [دیسکورد](https://discord.com/invite/8caJbAGNwY) ما.
+* #ravynOS-general:matrix.org - با [Element.io](https://app.element.io/#/room/%23ravynOS-general:matrix.org) به ما بپیوندید.
+
+[![بسته‌ها توسط Cloudsmith میزبانی می‌شوند](https://img.shields.io/badge/OSS%20hosting%20by-cloudsmith-blue?logo=cloudsmith&style=flat-square)](https://cloudsmith.com)
+
+---
+
+FreeBSD Source:
+---------------
+This is the top level of the FreeBSD source directory.
+
+FreeBSD is an operating system used to power modern servers, desktops, and embedded platforms.
+A large community has continually developed it for more than thirty years.
+Its advanced networking, security, and storage features have made FreeBSD the platform of choice for many of the busiest web sites and most pervasive embedded networking and storage devices.
+
+For copyright information, please see [the file COPYRIGHT](COPYRIGHT) in this directory.
+Additional copyright information also exists for some sources in this tree - please see the specific source directories for more information.
+
+The Makefile in this directory supports a number of targets for building components (or all) of the FreeBSD source tree.
+See build(7), config(8), [FreeBSD handbook on building userland](https://docs.freebsd.org/en/books/handbook/cutting-edge/#makeworld), and [Handbook for kernels](https://docs.freebsd.org/en/books/handbook/kernelconfig/) for more information, including setting make(1) variables.
+
+For information on the CPU architectures and platforms supported by FreeBSD, see the [FreeBSD
+website's Platforms page](https://www.freebsd.org/platforms/).
+
+Source Roadmap:
+---------------
+| Directory | Description |
+| --------- | ----------- |
+| bin | System/user commands. |
+| cddl | Various commands and libraries under the Common Development and Distribution License. |
+| contrib | Packages contributed by 3rd parties. |
+| crypto | Cryptography stuff (see [crypto/README](crypto/README)). |
+| etc | Template files for /etc. |
+| gnu | Commands and libraries under the GNU General Public License (GPL) or Lesser General Public License (LGPL). Please see [gnu/COPYING](gnu/COPYING) and [gnu/COPYING.LIB](gnu/COPYING.LIB) for more information. |
+| include | System include files. |
+| kerberos5 | Kerberos5 (Heimdal) package. |
+| lib | System libraries. |
+| libexec | System daemons. |
+| release | Release building Makefile & associated tools. |
+| rescue | Build system for statically linked /rescue utilities. |
+| sbin | System commands. |
+| secure | Cryptographic libraries and commands. |
+| share | Shared resources. |
+| stand | Boot loader sources. |
+| sys | Kernel sources (see [sys/README.md](sys/README.md)). |
+| targets | Support for experimental `DIRDEPS_BUILD` |
+| tests | Regression tests which can be run by Kyua.  See [tests/README](tests/README) for additional information. |
+| tools | Utilities for regression testing and miscellaneous tasks. |
```

Comment:
perhaps these should be in Persian as well?

Reply:
I see that other translations only translated first part of README file, and I did so.
By the way translating this part into Persian can make confusion and bi-directional problems. I think it's best to avoid it.

Reply:
Good point, thanks for pointing that out